src/lib/operators/materialize.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "abstract_read_only_operator.hpp"
+
+namespace opossum {
+
+/**
+ * @brief Materializes a table
+ *
+ * Forwards value columns. Turns all other columns into value columns.
+ *
+ * The operator does not copy the MVCC columns of the input table.
+ */
+class Materialize : public AbstractReadOnlyOperator {
+ public:
+  explicit Materialize(const std::shared_ptr<const AbstractOperator> in);
+
+  const std::string name() const final;
+  const std::string description(DescriptionMode description_mode) const final;
```

Comment:
Why are you overriding if the implementation is the same as in the base-base class?